javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+const heading = document.querySelector('h1');
+
+function countDown() {
+  if (count > 0) {
+    heading.textContent = count;
+    count--;
+  } else {
+    heading.textContent = '~Earth Beeelowwww Us ~';
+    clearInterval(intervalId);
+  }
+}
+
+let count = 4;
+
+const intervalId = setInterval(function () {
+  countDown(count);
```

Comment:
No argument here. You're calling `countDown(count)` but you don't add a parameter to the `countDown` function definition. Since `count` is defined globally, this doesn't need an argument